src/ScoutingMain.elm

```diff
@@ -246,11 +246,16 @@ update msg model =
 
                 verifier : Bool
                 verifier =
-                    (not <| List.member matchError [ Err "No such match", Err "Invalid match number" ])
-                        && stationError
+                    stationError
                         /= Err "No station"
```

Comment:
unclear

Reply:
but works so i will approve it

Reply:
You should insist more on these kind of comments. Code should be written for humans, so if something can be clearer it's worth the effort, especially now that you have time to make everything better.

Reply:
I changed it. tell me if it is still unclear.